src/test/kotlin/no/nav/arbeidsplassen/importapi/dto/PropertyEnumValidationTest.kt

```diff
@@ -37,4 +37,41 @@ class PropertyEnumValidationTest(private val propertyEnumValidation: PropertyNam
         val error = assertThrows<ImportApiError> { propertyEnumValidation.checkOnlyValidValues(errors) }
         assertEquals(ErrorType.INVALID_VALUE, error.type)
     }
+
+    @Test
+    fun validatePropertiesWithMultipleValues() {
```

Comment:
Lag gjerne en test på en eksisterende verdi for å bevise at det gamle fremdeles funker. Dvs f.eks kun "Søndag"

Reply:
Det var en route bakt inn i samme testfunksjon, men har splittet opp testen i flere mer eksplisitte tester i [ce58c73](https://github.com/navikt/pam-import-api/pull/73/commits/ce58c7351070c6d725265e15dae24deb51f13daf)